packages/app/src/app/pages/common/GithubIntegration/index.tsx

```diff
@@ -22,7 +22,7 @@ export const GithubIntegration: FunctionComponent<Props> = ({
 
   return (
     <Integration
-      bgColor="#4078c0"
+      bgColor="#0971f1"
```

Comment:
This one is tricky, the old one was based on the brand colors from GitHub. I'm fine with changing this, but we should keep it in mind cc @DannyRuchtie .

Reply:
This was asked by Danny